src/app/admin/invite/invite-form.tsx

```diff
@@ -36,7 +36,10 @@ export function InviteForm() {
         },
         onSuccess(info) {
             notifications.show({ message: `User invited successfully\nClerk ID: ${info.clerkId}`, color: 'green' })
-            studyProposalForm.setValues(initialValues())
+            // Keep the selected organization, reset other fields using initialValues
+            const oldOrgId = studyProposalForm.values.organizationId
+            studyProposalForm.reset()
+            studyProposalForm.setFieldValue('organizationId', oldOrgId)
```

Comment:
take a look at [the docs](https://mantine.dev/form/use-form/#values)

they propose using something like this 🤔 
```
// Sets initial values, used when form is reset
form.setInitialValues({ values: 'object' });
```